### PR TITLE
Only check for moving draggable in canvas renderer when map is draggable

### PR DESCRIPTION
**Pull request by klaftertief**
This is a fix for #3941

### CODE REVIEW

File: src/layer/vector/Canvas.js [hunk 1]
```diff
@@ -215,7 +215,7 @@ L.Canvas = L.Renderer.extend({
 	},
 
 	_onMouseMove: function (e) {
-		if (!this._map || this._map.dragging._draggable._moving || this._map._animatingZoom) { return; }
+		if (!this._map || (this._map.dragging._enabled && this._map.dragging._draggable._moving) || this._map._animatingZoom) { return; }
```
Comment (yohanboniface) [inline src/layer/vector/Canvas.js]: This becomes a bit hard to follow, I wonder if we should not have a method on the L.Draggable object to make this more readable.
Something like `this._map.dragging.moving()` that would do both checks.

Comment (yohanboniface) [inline src/layer/vector/Canvas.js]: Also, `this._map._dragging`, inheriting from `L.Handler` has a `enabled()` method that should be used here instead of private property.

**Comment by yohanboniface** [comment]
Good catch! See code comments :)

File: src/layer/vector/Canvas.js (hunk 1 above)
Comment (mourner) [inline src/layer/vector/Canvas.js]: I agree. Should be a simple change, but let's look if we can update similar places to use the new method.

Comment (klaftertief) [inline src/layer/vector/Canvas.js]: Yep. I'm happy to update the pull request later.

**Comment by klaftertief** [comment]
Ok, I just added a moving() method to the MapDrag handler, just like the moved() method there.
I didn't find any other occurrences of the `_moving` property, except in the `Draggable` class itself.
